releasenotes/notes/add-ndm-to-fips-mode-f0b6e379369c3f48.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
```

Comment:
Doublechecking if the formatting is the same between   `  - |` and `-`
I saw other PRs formatted like the following:
```
features:
  - |
    Add FIPS mode support for Network Device Monitoring products
```

Reply:
If I'm not mistaken,`|` is used to write multi lines strings in YAML so I believe both are working correctly in this specific case